myhpi/core/models.py

```diff
@@ -215,3 +215,12 @@ class AbbreviationExplanation(Model):
 
     def __str__(self):
         return self.abbreviation
+
+
+class MailingListsMenuItem(BasePage):
```

Comment:
this is quite specific. What do you think about adding a RedirectMenuItem instead on which the user can set a target? This would also allow for external links to be set in the menu, sounds like something one could probably need in the future anyways

Reply:
Good idea!